setup.py

```diff
@@ -28,7 +28,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     python_requires=">=3.9",
-    install_requires=["Django>=4.2", "python-dateutil>=2.7.0"],
+    install_requires=["Django>=4.2,<5.3", "python-dateutil>=2.7.0"],
```

Comment:
I think we don't need to limit the upper bound
```suggestion
    install_requires=["Django>=4.2", "python-dateutil>=2.7.0"],
```